src/main/groovy/com/twcable/grabbit/jcr/JCRNodeDecorator.groovy

```diff
@@ -25,6 +25,7 @@ import org.apache.jackrabbit.value.DateValue
 
```

Comment:
Name of this file is different from the class. JcrNodeDecorator/JCRNodeDecorator